app/controllers/assignments_controller.rb

```diff
@@ -62,6 +72,13 @@ def set_assignment
     @assignment = @organization.assignments.includes(:assignment_invitation).find_by!(slug: params[:id])
   end
 
+  def student_identifier_type_param
+    unless params.key?(:student_identifier_type) && params.key?(:student_identifier_type)
```

Comment:
This can just be

```ruby
return unless params.key?(:student_identifier_type) && params.key?(:student_identifier_type)
StudentIdentifierType.find_by(id: student_identifier_params[:id], organization: @organization)
```